pom.xml

```diff
@@ -12,7 +12,7 @@
 
   <groupId>com.puppycrawl.tools</groupId>
   <artifactId>checkstyle</artifactId>
-  <version>8.30</version>
+  <version>8.31-SNAPSHOT</version>
```

Comment:
Please rebase your branch on the lastest master. We should not see any changes that you did not personally make. I would review your commit before pushing to the server.